op-deployer/pkg/deployer/flags.go

```diff
@@ -114,6 +115,23 @@ var (
 		Usage:   "(optional) contract name matching a field within the input file",
 		EnvVars: PrefixEnvVar("CONTRACT_NAME"),
 	}
+	VerifierFlag = &cli.StringFlag{
+		Name:    VerifierFlagName,
+		Usage:   "contract verifier to use. options: etherscan (default), blockscout, custom",
+		EnvVars: PrefixEnvVar("VERIFIER"),
+		Value:   "etherscan",
+	}
```

Comment:
Suggestion: rename to `VerifierService` or `VerifierType` (or something similar) to make it more obvious what this is. `Verifier` is kind of vague